policy/beta/source_image.rego

```diff
@@ -42,6 +42,27 @@ _source_image_errors contains error if {
 	error := sprintf("Source image has no layers %q", [img])
 }
 
+# METADATA
+# title: Signed
+# description: Verify the source container image is signed.
+# custom:
+#   short_name: signed
+#   failure_msg: "%s"
+#   depends_on:
+#   - source_image.exists
+#
+deny contains result if {
+	some error in _source_image_sig_errors
+	result := lib.result_helper(rego.metadata.chain(), [error])
+}
+
+_source_image_sig_errors contains error if {
+	some img in _source_images
+	info := ec.sigstore.verify_image(img, lib.sigstore_opts)
+	some raw_error in info.errors
+	error := sprintf("Image signature verification failed for %s: %s", [img, raw_error])
```

Comment:
I wonder why the code coverage check didn't fail for this line, I don't see a test reaching this

Reply:
test_missing_signature does. See the expected error messages [here](https://github.com/enterprise-contract/ec-policies/pull/946/files#diff-24cbb169deacc3371a03a72775c6e0c70311d24131c4d1e9c93166dbb01c1d99R249-R261).

Reply:
Thanks, missed this somehow...